### ISSUE TITLE
Drywallow Vicejaw (skinning)

### ISSUE THREAD

**Issue by Firestoat** [slowtorta/turtlewow-bug-tracker#1394]
The Drywallow Vicejaw mob located in Dustwallow Marsh is currently not giving a player a leather based drop from Skinning skill about 50% of the time. Oddly enough, the other crockolisk style mobs in the zone aren't having this issue from what I could tell - only the Vicejaws have the broken loot. This is probably a database issue associated with the mob.

In Game Mob Image -- https://postimg.cc/R6b34fb9/1dba27ca

TurtleWoW Mob Link -- https://database.turtle-wow.org/?npc=4342

WoWHead Classic Link -- https://classic.wowhead.com/npc=4342/drywallow-vicejaw

**Comment by slowtorta** [comment]
It is working as intended! The value is the original one, it's expected that not all the mobs drop a skin.

**Comment by Firestoat** [comment]
Thank you very much for looking into this. It appears there may be some confusion involving the issue I reported. In old retail WoW from 2004 through WoTLK pre cataclysm, every single mob that was tagged as being able to be skinned always gave a skinning loot drop of some sort when complete. This could be a form of leather, a hide, a dragonscale, or a quest drop associated with skinning in addition to the mob's normal leather / skinning reward.

This zone is the only one I've encountered on this private server emu core that has the situation of roughly a 50% chance of not getting a skinning related loot drop, and only with the mobs I did a report on. If the Emu core has a shortcoming that precludes a fix for the situation, that's understandable.
